packages/netlify-cms-core/src/actions/config.js

```
@@ -54,24 +56,50 @@ export function applyDefaults(config) {
         map.setIn(['slug', 'sanitize_replacement'], '-');
       }
 
+      const locales = map.get('locales');
       // Strip leading slash from collection folders and files
       map.set(
         'collections',
         map.get('collections').map(collection => {
           const folder = collection.get('folder');
           if (folder) {
+            const fields = collection.get('fields');
+            const identifier_field = selectIdentifier(collection);
+            if (locales && fields && collection.get('multi_content')) {
+              // add locale fields
+              collection = collection.set(
+                'fields',
+                fromJS(addLocaleFields(fields.toJS(), locales.toJS())),
+              );
+
+              // remove path for same or different folder config
+              if (DIFF_FILE_TYPES.includes(collection.get('multi_content'))) {
+                collection = collection.delete('path');
+              }
+              // add identifier config
+              collection = collection.set(
+                'identifier_field',
+                `${locales.first()}.${identifier_field}`,
+              );
+            }
+
             if (collection.has('path') && !collection.has('media_folder')) {
               // default value for media folder when using the path config
               collection = collection.set('media_folder', '');
             }
+
             if (collection.has('media_folder') && !collection.has('public_folder')) {
               collection = collection.set('public_folder', collection.get('media_folder'));
             }
+
             return collection.set('folder', trimStart(folder, '/'));
           }
 
           const files = collection.get('files');
           if (files) {
+            // remove multi_content config if set
+            collection = collection.delete('multi_content');
```

Can we forbid it in the schema instead of silently ignoring it?